instrumentation/jboss-logmanager-mdc-1.1/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.jboss.logmanager")
+    module.set("jboss-logmanager")
+    versions.set("[1.1.0.GA,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  library("org.jboss.logmanager:jboss-logmanager:1.1.0.GA")
+
+  compileOnly(project(":instrumentation-appender-api-internal"))
+
+  // ensure no cross interference
+  testInstrumentation(project(":instrumentation:java-util-logging:javaagent"))
```

Comment:
I think that's probably not needed in case of this instrumentation; there's no risk of interfering with JUL because JUL has no concept of MDC.

Reply:
Done